word_scripts/wordfind_site.py

```diff
@@ -0,0 +1,26 @@
+import requests
+from bs4 import BeautifulSoup
+
+def wordFindSite ():
```

Comment:
In Python, the common convention is [underscore_case](http://programmers.stackexchange.com/questions/27264/naming-conventions-camelcase-versus-underscore-case-what-are-your-thoughts-ab), so ex. this would be word_find_site

Also this could have a better name for this function like parse_word_find_site or extract_from_word_find_site. Method names are usually begin with verbs to describe what it's doing.